lib/src/model/container.dart

```diff
@@ -74,7 +75,10 @@ abstract class Container extends ModelElement with TypeParameters {
   /// This is only used in mustache templates.
   bool get publicInheritedInstanceMethods => false;
 
-  Iterable<Operator> get publicInheritedInstanceOperators => [];
+  /// Whether all instance operators are inherited.
+  ///
+  /// This is only used in mustache templates.
```

Comment:
it's not restricted to them though, so probably this little comment can be deleted.

Reply:
I've removed it from this getter and the related nearby ones as well.